packages/app/src/runner/aut-iframe.ts

```diff
@@ -20,6 +20,9 @@ export class AutIframe {
   debouncedToggleSelectorPlayground: DebouncedFunc<(isEnabled: any) => void>
   $iframe?: JQuery<HTMLIFrameElement>
   _highlightedEl?: Element
+  // Cache the blank page content to avoid repeated generation
+  private _cachedTestIsolationContent?: string
+  private _cachedInitialContent?: string
```

Comment:
Our initial blank page and the test isolation page that shows in between each test is just a string, HTML string. This can be cached upfront so we don't read this string through over and over. Likely not a huge win, but this can be shown a lot with a lot of tests.